.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+**Summary:** 
+
+Summarize your issue in one sentence (what goes wrong, what did you expect to happen)
+
+**Steps to reproduce:** 
+
+How can we reproduce the issue, or where do we need to look in the data to find the problem (e.g., stops.txt stop_desc)?  Can we see this in an app?
+
+**Expected behavior:** 
+
+What did you expect the data to look like?
+
+**Observed behavior:** 
+
+What does the data look like instead?  Describe your issue in detail here.
+
+**Feed URLs:** 
```

Comment:
Would globally call that `sources`, since we'll also have link to schedule on website (when mismatch between GTFS schedule and website schedule)